local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Sun Mar 21 11:50:51 CET 2021
+sdk.dir=/home/cpittet/Android/Sdk
```

Comment:
Why did you push your local.properties?

Reply:
Agree, you should add it into your .gitignore

Reply:
Oups, right